src/components/Sidebar/style.scss

```
@@ -0,0 +1,44 @@
+@import '../../styles/constants';
+
+.sidebar {
+  display: flex;
+  height: fit-content;
+  margin: 20px 0 20px 20px;
+  background-color: $light-purple;
+  padding: 15px;
+  border: 3px solid $light-gray;
+  box-shadow: 0 0 0 1px $gray,
+    inset 0 0 0 1px $gray,
+    1px 2px 0 2px $dark-gray;
+  border-radius: 10px;
+  align-items: center;
+
+  &__header {
+    img {
+      width: 45px;
+      height: 45px;
+      border-radius: 50px;
+      object-fit: cover;
+    }
+
+    text-align: center;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
```

In situations like this one we first put everything of the &__header on top and then continue with the other selectores
```suggestion
    text-align: center;
    display: flex;
    align-items: center;
    justify-content: center;
    
    img {
      width: 45px;
      height: 45px;
      border-radius: 50px;
      object-fit: cover;
    }
  }
```